sycl/test-e2e/Matrix/joint_matrix_bfloat16_impl.hpp

```diff
@@ -85,13 +82,6 @@ bfloat16 B[MATRIX_K / 2][MATRIX_N * 2];
 float C[MATRIX_M][MATRIX_N];
 float D[MATRIX_M][MATRIX_N];
 
-float make_fp32(bfloat16 x) {
-  unsigned int y = *((int *)&x);
-  y = y << 16;
-  float *res = reinterpret_cast<float *>(&y);
-  return *res;
-}
-
 void matrix_multiply_ref(int *A_mem, int *B_mem, int *C_mem, int M, int N,
```

Comment:
Sounds good, make sure this function also goes to common.hpp. But since here, we assume data are VNNIed, this should be called something like: matrix_multiply_vnni_ref